features/steps/browser_steps.rb

```diff
@@ -0,0 +1,58 @@
+When("I navigate to the URL {string}") do |path|
+  $driver.navigate.to get_test_url path
+end
+
+When("the test should run in this browser") do
+  wait = Selenium::WebDriver::Wait.new(timeout: 10)
+  wait.until {
+    $driver.find_element(id: 'bugsnag-test-should-run') &&
+    $driver.find_element(id: 'bugsnag-test-should-run').text != 'PENDING'
+  }
+  skip_this_scenario if $driver.find_element(id: 'bugsnag-test-should-run').text == 'NO'
+end
+
+When("I let the test page run for up to {int} seconds") do |n|
+  wait = Selenium::WebDriver::Wait.new(timeout: n)
+  wait.until {
+    $driver.find_element(id: 'bugsnag-test-state') &&
+    $driver.find_element(id: 'bugsnag-test-state').text == 'DONE'
+  }
+end
+
+When("the exception matches the {string} values for the current browser") do |fixture|
+  err = get_error_message(fixture)
+  steps %Q{
+    And the exception "errorClass" equals "#{err['errorClass']}"
+    And the exception "message" equals "#{err['errorMessage']}"
+  }
+end
+
+Then(/^the request is a valid browser payload for the error reporting API$/) do
```

Comment:
The checks here are good, was there a reason why this needs to be separate from the standard error reporting API step in mazerunner? There may be other notifiers which don't send an API key in the headers.